.github/workflows/S3-deploy.yml

```diff
@@ -0,0 +1,38 @@
+name: S3-deploy
+
+on:
+  push:
+    branches:
+      - "development"
+      - "react-S3-action"
+  pull_request:
+    branches:
+      - "development"
+      - "react-S3-action"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [15.x]
+
+    steps:
+      - uses: actions/checkout@v1
+      - run: npm install
+        working-directory: ./frontend/floop_feedback
+      - run: npm install mini-css-extract-plugin@~2.4.5
```

Comment:
This is exactly what I had to install when I mentioned I was having trouble building the app last night!